packages/twenty-server/src/engine/core-modules/user/user.entity.ts

```diff
@@ -34,6 +34,7 @@ registerEnumType(OnboardingStatus, {
   unique: true,
   where: '"deletedAt" IS NULL',
 })
+@Index('IDX_USER_ID_DELETED_AT', ['id', 'deletedAt'])
```

Comment:
I don't think this one is useful but might be wrong. This will be a composite index, indexed on id which is the primary key (btree indexed) so the improvement on btree will be negligeable